src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -1237,6 +1243,37 @@ public static String toString(final Object obj, final String nullStr) {
         return obj == null ? nullStr : obj.toString();
     }
 
+    /**
+     * <p>Validate that the specified argument object is neither {@code null}
```

Comment:
Please close HTML tags. The FIRST sentence in a Javadoc does not need to be in a paragraph, it is implied.